Sources/SentryCrash/Recording/Tools/SentryHook.h

```diff
@@ -0,0 +1,38 @@
+#ifndef SENRTY_HOOK_h
+#define SENRTY_HOOK_h
+
+#include "SentryCrashThread.h"
+#include <stdint.h>
+
+#define MAX_BACKTRACE_FRAMES 128
```

Comment:
`l`: What happens if there are more frames? Are they just cut?

Reply:
Yes.
Its the same number as in the native sdk: https://github.com/getsentry/sentry-native/blob/1b151d10fdb4b1fa2e4440cff4a436d6186fc862/src/backends/sentry_backend_inproc.c#L21
Also SentryCrash has its own limit for `SelfThread` which I think is even lower.